packages/dendron-plugin-views/src/components/DendronSideGraphPanel.tsx

```diff
@@ -0,0 +1,16 @@
+import { ideHooks, ideSlice } from "@dendronhq/common-frontend";
+import { DendronComponent } from "../types";
+import DendronGraphPanel from "./DendronGraphPanel";
+
+/**
+ * Wrapper component around DendronGraphPanel
+ * @param props
+ * @returns DendronGraphPanel component with ide.isSidePanel set to true
+ */
+const DendronSideGraphPanel: DendronComponent = (props) => {
+  const ideDispatch = ideHooks.useIDEAppDispatch();
+  // set side panel to true
+  ideDispatch(ideSlice.actions.setIsSidePanel(true));
```

Comment:
instead of making this a state property, can we just have this as part of the `props` for `DendronGraphPanel`? this way we avoid an unnecessary dispatch